models/project/project.go

```diff
@@ -371,3 +308,21 @@ func (u *ProjectStruct) FetchGithub() (bool, error) {
 
 	return true, nil
 }
+
+func (u *ProjectStruct) OauthToken() (string, error) {
```

Comment:
__[golint]__
* [golint] exported method ProjectStruct.OauthToken should have comment or be unexported